.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "resources"]
 	path = resources
 	url = git://github.com/w3c/testharness.js.git
```

Comment:
Not in this patch (yet) but note that the above should instead be `https://github.com/w3c/w3c/testharness.js.git` (the `https:` URL instead of the `git:` one)